src/lib.rs

```diff
@@ -228,6 +250,17 @@ pub struct RangeStep<A> {
     rev: bool,
 }
 
+#[cfg(rustc_1_28)]
+impl<A> RangeBounds<A> for RangeStep<A> {
```

Comment:
Do you _need_ this for the step variants? They make less sense to me for generic bounds since they are not a continuous range. The standard library equivalent `StepBy<Range<_>>` certainly doesn't have this, only iterator traits.

Reply:
Jup, that makes sense :) will correct that